src/SfxWeb/src/app/views/replica/essentials/essentials.component.ts

```diff
@@ -28,19 +28,32 @@ export class EssentialsComponent extends ReplicaBaseControllerDirective {
   }
 
   refresh(messageHandler?: IResponseMessageHandler): Observable<any> {
-    return forkJoin([
-      this.replica.health.refresh(messageHandler),
-      this.replica.detail.refresh(messageHandler).pipe(map(() => {
-        if (!this.isSystem) {
+  const safeHealthRefresh$ = this.replica.health.refresh(messageHandler).pipe(
+    catchError(error => {
```

Comment:
was there a concern around if one of these calls fails? Right now we dont update the view if one is unavailable. I am more thinking we might need a cleaner solution to partially update the state like essential items.

Reply:
oh, could the entry potentially not exist in the health store? One way might be only updating the essential items that are relevant in their respective call.